src/gConsent/util/issueAccessVc.ts

```diff
@@ -210,13 +211,13 @@ export async function issueAccessVc(
     hasConsent &&
     vcBody.credentialSubject.hasConsent.forPersonalData.length <= 0
   ) {
-    throw new Error("There are no resources in the access request!");
+    throw new AccessGrantError("There are no resources in the access request!");
```

Comment:
```suggestion
    throw new AccessGrantError("There are no resources in the access request");
```
For consistency with the related message but then there is also a question of consistency throughout. Should error messages end with punctuation or not?

Reply:
I don't have a strong opinion on this. I would say they should not, because some of them will end wrapping another error message sent by an underlying library, which we do not control and cannot know whether it ends with punctuation or not.